dev/Dockerfile.dev

```diff
@@ -1,9 +1,8 @@
-FROM phusion/baseimage:0.11
+FROM registry.tld/ruby-fips-base-image-phusion:1.0.0
```

Comment:
Did you consider or attempting basing this on `registry.tld/ruby-fips-base-image-ubuntu:1.0.0` to get this on a shared based image instead?

Reply:
The plan is to consume this image from Dockerhub

Reply:
I meant did you consider not creating two base images:
- `ruby-fips-base-image-ubuntu:1.0.0`
- `ruby-fips-base-image-phusion:1.0.0`

And instead only use one in all places, replacing the use of `phusion` in the OSS repo: `ruby-fips-base-image-ubuntu:1.0.0`.

That would reduce the maintenance overhead of two base images just for a separate dev environment (and it's not clear the dev environment needs to be `phusion` based).

Reply:
Currently, OSS is using the Conjur base image and appliance the phusion base image.
Uniting them to one image is required but it is out of scope for this task.

Reply:
Hi Nahum, 

I'm not referring to the appliance. I understand that it still uses the `phusion`, and needs to for a variety of reasons. 

I'm referring to this `Dockerfile.dev` that we're commenting on for the dev environment in OSS. It should be able to use the `ruby-fips-base-image-ubuntu:1.0.0` base image.

Reply:
Micah hi,
Sorry I missed your point twice. I absolutely agree. I've tried to use the ubuntu but faced problems therefore left it. 
It is out of scope for this assignment to change the base image. Should be addressed in a different task

Reply:
Sounds good! If it was an easy swap, then it would be worth doing. I agree that if it's much more work, it should be separate. Thanks! :)